smact/property_prediction/tests/test_base_predictor.py

```
@@ -0,0 +1,185 @@
+            predictor._validate_compositions([], validate_smact=False)
+
+    def test_smact_validation(self):
+        """Should validate compositions using SMACT when enabled."""
+        # Test that validation runs without crashing on valid composition
+        # Skip if SMACT has compatibility issues (Python version, etc.)
+        try:
+            from smact.screening import smact_validity
+
+            # First check if smact_validity works at all
+            smact_validity("NaCl")
+        except Exception as e:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)